packages/nouns-sdk/src/image/svg-builder.ts

```diff
@@ -84,3 +85,48 @@ export const buildSVG = (
 
   return `${svgWithoutEndTag}</svg>`;
 };
+
+/**
+ * Given the RLE data of an individual part, palette colors, and a background color, build an SVG image.
+ * @param partData The RLE part datas
+ * @param paletteColors The hex palette colors
+ */
+export const buildPartSVG = (
```

Comment:
Nit: should be able to accomplish the same thing with `buildSVG`, right? Maybe just make `bgColor` optional and skip the background fill if it's not passed.

Reply:
`buildSVG` accepts and iterates through the part data of multiple traits. and this new function accepts individual part data. 
they may be able to be combined, but i believe it'll require some additional changes beyond the `bgColor` conditional, and i didn't want to mess with this critical function too much. but i'm happy to combine them into if that makes more sense.

Reply:
Yeah, doesn't quite seem necessary to have an entirely different function when you could achieve the same by doing:

```typescript
buildSVG([{ data }], palette);
// rather than
buildPartSVG(data, palette)
```

Reply:
yes! that's a much simpler solution. thanks for the guidance here @solimander.

this has been resolved in https://github.com/nounsDAO/nouns-monorepo/pull/595/commits/61b814dd7e5570898dd9eb1b8f9214586758637d